rocblas/PKGBUILD

```diff
@@ -1,51 +1,57 @@
 # Maintainer: Markus Näther <naether.markus@gmail.com>
 pkgname=rocblas
-_pkgver=3.3.0
-pkgver="$_pkgver"
-pkgrel=3
-pkgdesc="Next generation BLAS implementation for ROCm platform"
+pkgver=3.3.0
+pkgrel=4
+pkgdesc='Next generation BLAS implementation for ROCm platform'
 arch=('x86_64')
-url="https://github.com/ROCmSoftwarePlatform/rocBLAS"
-license=('custom:NCSAOSL')
-depends=('hcc' 'hip')
-makedepends=('cmake' "hcc>=$pkgver" 'python' 'boost' "comgr>=$pkgver" 'rocminfo' 'hsa-ext-rocr')
-source=("https://github.com/ROCmSoftwarePlatform/rocBLAS/archive/rocm-$_pkgver.tar.gz")
-sha256sums=('aaa8f0479202bdbe94d2ec5655a76055656f743b0d36816501cb84a533084034')
+url='https://rocmdocs.amd.com/en/latest/ROCm_Libraries/ROCm_Libraries.html#rocblas'
+license=('MIT')
+depends=('hip-hcc')
+makedepends=('cmake' 'hcc' 'python' 'git' 'perl-file-which' 'llvm-amdgpu')
+_git='https://github.com/ROCmSoftwarePlatform/rocBLAS'
+source=("$pkgname-$pkgver.tar.gz::$_git/archive/rocm-$pkgver.tar.gz"
+        'hcc_workweek.patch')
```

Comment:
I believe there might be a way around this patch. Can you try building our `hcc` package in clean chroot with `git` as a dep and then rebuild `hip-hcc` without the patch? I have a feeling it might work, and I don't want to merge the patch in if we don't have to.

Reply:
I don't like the patch either. Your idea sounds good. I'll give it a try.

Reply:
Thank you so much for your efforts, we really appreciate it. I can just feel the moment when we get the stack working completely!

Reply:
I got rid of the patch by separately downloading Tensile. `__hcc_workweek__` is then patched similar to what we do for `hip-hcc` at the moment.

Regarding your idea: I tried to build `hcc-git` but the package() step fails with an internal linking issue. I then tried to use the `3.3.0` tags. With that compiler (so basically our `hcc` but build from the cloned repo instead of the downloaded archives) and a rebuilt `hip-hcc` I was not able to build `rocblas`. With the `hcc` package from `arch4edu` it works. I've no idea what's going wrong.

Reply:
Ah sorry for not being clear, I mean building regular `hcc` but with `git` as a `depends`, rather than `hcc-git`.

Reply:
In lieu of getting something working out to the masses I'll merge this in with the patch. Down the road we should try to have a patch tracker so that we can keep track of our local patches from the official ROCm repos.

Reply:
I totally agree with you. I really would like to try to compile `hcc` with `git` but I don't get it working. We'll have to revisit all packages anyway at some point when AMD is deprecating `hcc` and uses the clang/VDI backend: https://github.com/ROCm-Developer-Tools/ROCclr